### PR TITLE
Release/0.22.0

### PR DESCRIPTION
**Pull request by loick**

### CODE REVIEW

File: src/textField/story.tsx [hunk 1]
```diff
@@ -115,7 +115,6 @@ stories.add(
 stories.add(
   'input with autofocus',
   withInfo('TextField with autofocus')(() => {
```
Comment (loick) [inline src/textField/story.tsx]: broke during the npm publish

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- **[...]**
+
+# v0.21.0 (11/02/2019)
```
Comment (Bidjii) [inline CHANGELOG.md]: `v0.22.0` I guess

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
 # Unreleased
 
 - **[FIX]** Check icon animation not played correctly on Safari
```
Comment (pith) [inline CHANGELOG.md]: Can you move it in the `# v0.22.0` section ?

Comment (loick) [reply CHANGELOG.md]: It was not rolled out in the V0.22

**Pull request merged**